brambling/templates/brambling/event/organizer/update.html

```diff
@@ -295,6 +296,8 @@ <h4 class="panel-title">Payment</h4>
 
 {% block javascripts %}
 	{{ block.super }}
+	<script type="text/javascript" src="{% static 'zenaida/js/bootstrap/tab.js' %}"></script>
```

Comment:
I think it used to be for a thing which was then removed. Which is maybe what you saw.